pkg/virt-controller/watch/vm.go

```diff
@@ -58,104 +58,108 @@ func NewVMController(vmService services.VMService, recorder record.EventRecorder
 				logger.Error().Reason(err).Msg("Deleting VM target Pod failed.")
 			}
 			logger.Info().Msg("Deleting VM target Pod succeeded.")
-		} else if vm.Status.Phase == "" {
-			// Schedule the VM
-			vmCopy := v1.VM{}
+			return true
+		}
 
-			// Deep copy the object, so that we can safely manipulate it
-			model.Copy(&vmCopy, vm)
-			logger := logging.DefaultLogger().Object(&vmCopy)
+		if vm.Status.Phase == "" {
+			scheduleVM(vm, vmService, queue, key, restClient)
+			return true
+		}
+		return true
+	})
+}
+func scheduleVM(vm *v1.VM, vmService services.VMService, queue workqueue.RateLimitingInterface, key interface{}, restClient *rest.RESTClient) {
+	vmCopy := copyVM(vm)
+	logger := logging.DefaultLogger().Object(&vmCopy)
+	defaultGraphicsNetworking(vm)
+	if err := vmService.StartVM(&vmCopy); err != nil {
```

Comment:
We should really rename these methods, the name I chose is just completely wrong, since it does don't start a VM. Something like `StartVMTargetPod()` maybe. But maybe you do that later on already ...